### PR TITLE
Create ruby.yml

### PR DESCRIPTION
**Pull request by armins**

### CODE REVIEW

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+
```
Comment (whotwagner) [inline .github/workflows/ruby.yml:21]: Would you mind to add tests for `overcommit --run` and bakerman?

**Review by whotwagner — CHANGES_REQUESTED** [review]
Do you mind to add tests for `overcommit --run` and bakerman?

File: .github/workflows/ruby.yml (hunk 1 above)
Comment (armins) [reply .github/workflows/ruby.yml:21]: done

Comment (armins) [reply .github/workflows/ruby.yml:21]: did you mean breakman? this is run by overcommit already

**Review by whotwagner — APPROVED** [review]

File: .github/workflows/ruby.yml (hunk 1 above)
Comment (whotwagner) [reply .github/workflows/ruby.yml:21]: not with run. breakman is executed just as a pre-push-action in overcommit which is not executed with `--run`

Comment (armins) [reply .github/workflows/ruby.yml:21]: ok, thanks for clarification. will add ...

**Pull request merged**